files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/default_tool.yml

```
@@ -15,7 +15,14 @@ tools:
     scheduling:
       reject:
         - offline
-    rules: []
+    rules:
+      - if: "True"
+        execute: |
+          training_roles = [r.name for r in user.all_roles() if not r.deleted and "training" in r.name]
+          training_expr = " || ".join(['(GalaxyGroup == "%s")' % role for role in training_roles])
+          training_labels = '"' + ", ".join(['%s' % role for role in training_roles]) + '"'
```

```suggestion
          training_labels = '"' + ", ".join(training_roles) + '"'
```

noticed while working on tiaas paper

---

Thanks https://github.com/usegalaxy-eu/infrastructure-playbook/pull/463